web/src/beta/components/fields/SelectField/index.tsx

```diff
@@ -51,7 +53,7 @@ const SelectField: React.FC<Props> = ({
   }, [options, value]);
 
   return (
-    <Property name={name} description={description}>
+    <Property name={name} description={description} className={className}>
```

Comment:
@KaWaite is it okay/necessary to have class here? I thought we are avoiding to use them?

Reply:
I used it because I wanted to modify the width as styled component and to avoid modify the style here immediately as we are using it widely in system

Reply:
@nina okay approved